### ISSUE TITLE
autosummary's template option doesn't work as documented

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#334]
The documentation at http://sphinx.pocoo.org/ext/autosummary.html says that the autodoc stubs generated by the autosummary extension can be customised by providing a custom template pointed to by the template option. However, when I try to do that, I get the following error:

```
#!text

$ bin/sphinx-build 
Running Sphinx v0.6.4
No builder selected, using default: html
generating autosummary for: /home/thomas/.../doc/modules.txt
building [html]: targets for 9 source files that are out of date
updating environment: 9 added, 0 changed, 0 removed
reading sources... [100&#37;] index                                                                                                                                                                                
/home/thomas/Check-outs/.../modules.txt:5: (ERROR/3) Error in "autosummary" directive:
unknown option: "template".

.. autosummary::
    :toctree: ./
    :template: autosummary-module
```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/334
- Originally reported by: Anonymous
- Originally created at: 2010-02-14T15:12:52.129

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2010-02-21 20:48:07+00:00_

Thanks for reporting.  This option will only be available as of Sphinx 1.0 (which is not released yet) -- I've fixed the 0.6 docs in <<cset b81b7a046328dff1cac28b9b35920f4636a8465f>>.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2013-03-31 13:57:35+00:00_

Removing version: 0.6.3 (automated comment)
